src/components/PullRequestList.tsx

```diff
@@ -38,47 +61,107 @@ export interface PullRequestListProps {
   emptyMessage: string;
   mutingConfiguration: "allow-muting" | "allow-unmuting" | "none";
   newCommitsNotificationToggled: boolean | null;
+  onlyDirectRequestsToggled: boolean | null;
```

Comment:
This is adding quite a few props to `PullRequestListProps` for a smaller part of the UI. I think we may want to extract it all into a new component, then pass it as a single prop of type `ReactNode` here. This pattern would also be useful for the existing `newCommitsNotificationToggled` prop.

I'm happy to merge your PR as-is and make the change myself, unless you specifically feel like taking it on.

Reply:
would you be able to do the refactor? feel free to do it in this PR directly before merge.